### ISSUE TITLE
Kit NBT tags broken

### ISSUE THREAD

**Issue by KaspianDev** [EssentialsX/Essentials#3523]
### Information

**Full output of `/ess version`**: Happens on latest build EssentialsX-2.18.0.5.jar

```
<!-- Run the command in your console, and copy and paste the whole output here -->
```

**Server log**: 

**EssentialsX config**: 

### Details

**Description**  

**Steps to reproduce**  

**Expected behavior**  

**Screenshots**

**Comment by KaspianDev** [comment]
Description: NBT tags are not being saved in kits correctly
Steps to reproduce: Create a kit with plugin https://www.spigotmc.org/resources/armor-fairy-yijiki-phantom-traveller-yeti-more-custom-sets-headdatabase-support.56616/ and give it yourself

**Comment by JRoy** [comment]
Duplicate, solved by #3248
